src/TypeMatcher/ReferenceEntityAttribute/ReferenceEntityAttributeTypeMatcher.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Synolia\SyliusAkeneoPlugin\TypeMatcher\ReferenceEntityAttribute;
+
+use Synolia\SyliusAkeneoPlugin\Exceptions\UnsupportedReferenceEntityAttributeTypeException;
+
+final class ReferenceEntityAttributeTypeMatcher
```

Comment:
Interface is missing ?

Reply:
same as the ProductReferenceEntityAttributeValueValueBuilder comment